httpfuzzerprocessor/filtersResultWithStringOnBodyResponse.py

```diff
@@ -0,0 +1,52 @@
+# @author Timothée Ruffenach
+# Version 1.0
+# filters the fuzzing result with a string.
+
+from javax.swing import JOptionPane
+
+
+# global variable
+init = False
+entry = ""
+isCheck = False
+
+# Called after injecting the payloads and before forward the message to the server.
+def processMessage(utils, message) :
+    global number,payloads
+    if not init:
+        initialise()
+
+
+def initialise():
+    global init,entry,isCheck
+    
+    entry = ""
+
+    # ask stings to find
+    while entry == "":
+        entry = getString("what character string do you want to find ?")
```

Comment:
```suggestion
        entry = getString("What character string do you want to find ?")
```

Reply:
I made the changes and make correction of other sentence